nixos/modules/system/boot/networkd.nix

```diff
@@ -94,7 +94,7 @@ let
   checkNetwork = checkUnitConfig "Network" [
     (assertOnlyFields [
       "Description" "DHCP" "DHCPServer" "IPForward" "IPMasquerade" "IPv4LL" "IPv4LLRoute"
-      "LLMNR" "MulticastDNS" "Domains" "Bridge" "Bond" "IPv6PrivacyExtensions"
```

Comment:
So this breaks "old" configurations?

Shouldn't we have a change log entry for that? Or even better support (time limited) backwards compatibility?

Reply:
yes, it breaks old configurations that uses those attributes before.

As it's currently not the default for network management, plus the module is already pretty huge, I'd be more in favor of breaking/changing this for unstable/18.09 (with a changelog entry), than introducing a compability shim that blows up the code even more.